### PR TITLE
doc: improve getting-started/smr-disk.md 1st Sect

### PR DESCRIPTION
**Pull request by zdover23**
This PR makes minor but crucial syntactical changes to
the English in the front matter of the "Getting Started
with SMR Hard-Disks" chapter of the Getting Started Guide.

It might be that this little section means to say:
1. Drive Managed Interfaces are irrelevant to ZonedStorage.
2. Host Aware devices will work with ZonedStorage, but
   ZonedStorage treats them the way that it treats Host
   Managed devices (which is to say that it does not support
   random write capabilities, but instead respects the strong
   sequential write constraints imposed by the Zoned Block
   Interface).

If this list of two things does indeed preserve the intent of
this section, then I might return in the future to recast
this information more pithily. I set this comment down in
print here as a note to a future version of me, who will be
equipped with a more complete understanding of the interface
implementations and the proper manner of their explanation to
the ignorant.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/getting-started/smr-disk.md [hunk 1]
```diff
@@ -11,33 +11,35 @@ technology can have different interface implementations. This results in
...
+  identify and control the device's zones. This interface has two different 
+  variations, or model implementations (See [SMR Interface Implementations](../introduction/smr#smr-interface-implementations)). Those two model implementations are:
+
+	* **Host Aware:** This zone model offers the convenience and
+	  flexibility of Drive Managed disks (for example, it provides 
+          random write capabilities) and also supports the full set of 
+          zone commands defined in the ZBC and the ZAC standards. Host 
+          Aware disks can support both the regular block device 
+          abstraction ("regular disk") and the zoned block device 
+          abstraction.
```
Comment (damien-lemoal) [inline docs/getting-started/smr-disk.md:30]: Weird indentation... Mixed up spaces and tabs ?

Comment (zdover23) [reply docs/getting-started/smr-disk.md]: I've just looked at this carefully, and the only thing that I can guess is that you're not keen on vim's auto-indentation (gq) in Visual Mode. So I've manually added carriage returns and spaces in order to make this consistent with itself.

It built with no problems locally both ways.

Comment (zdover23) [reply docs/getting-started/smr-disk.md]: This looks fine in the repository when you examine the file with a text editor, but it renders with the weird spacing here in Github.

This is weird.

Comment (zdover23) [reply docs/getting-started/smr-disk.md]: Using "set syntax=whitespace" in vim, I seem to have corrected the strange indentation. However, I remain uncertain of exactly what changed. The highlighted syntax looked the same before the change as it did after the change.

Despite my continuing confusion in this connection, I think this is repaired.

**Pull request merged**